index.html

```diff
@@ -0,0 +1,13 @@
+Tiffany D Cooper
+"About"
+I love to travel and watch my son play basketball. I was a mainframe developer for 15 years.
+I am also a travel agent and I book vacations for others.
+"Experience"
+Total Systems Services
+Alfa Life Insurance
+Alabama Department of Education
+Travel agent
+"Connect"
+https://github.com/TiffanyD8/intro-to-programming-dorado/branches
```

Comment:
I don't think you meant to include the last line with the C drive file path. 
For your github link, I would suggest just linking to your home page: https://github.com/TiffanyD8